src/Network/Network/help/New-AzPrivateEndpoint.md

```diff
@@ -24,14 +24,18 @@ The **New-AzPrivateEndpoint** cmdlet creates a private endpoint.
 ## EXAMPLES
 
 ### 1: Create a private endpoint
```

Comment:
Is this supposed to be "Example 1"?

Reply:
Good catch. I've corrected this along with performing a more thorough overall cleanup of the file to include standardizing the markdown and clarifying some of the content.